auth/parsing/src/main/java/org/wildfly/security/auth/parsing/ElytronXmlParser.java

```diff
@@ -127,11 +136,12 @@
 import org.wildfly.security.ssl.X509RevocationTrustManager;
 
 /**
- * A parser for the Elytron XML schema.
+ * A parser for the Elytron XML schema. A provider of the ElytronXmlParserService.
  *
  * @author <a href="mailto:david.lloyd@redhat.com">David M. Lloyd</a>
  */
-public final class ElytronXmlParser {
```

Comment:
Hello @ashley-abdelsayed98 , is it possible to not remove old class of ElytronXmlParser and rather deprecate it?

Reply:
I've kept the old ElytronXmlParser and just deprecated it